lib/buffer.js

```diff
@@ -65,9 +65,72 @@ function Buffer(arg, encoding) {
   return fromObject(arg);
 }
 
+/**
+ * Functionally equivalent to Buffer(arg, encoding) but throws a TypeError
+ * if value is a number.
+ * Buffer.from(str[, encoding])
+ * Buffer.from(array)
+ * Buffer.from(buffer)
+ * Buffer.from(arrayBuffer)
+ **/
+Buffer.from = function(value, encoding) {
+  if (typeof value === 'number')
+    throw new TypeError('"value" argument must not be a number');
+
+  // Slightly less common case.
```

Comment:
This comment doesn't make sense. So the function only has slightly and unusual cases?